backend/handlers/user_handler.py

```diff
@@ -29,6 +29,16 @@ def get_invites(user_email):
 
     return invites
 
+def get_requests(user_key):
+    """Query to return the list of requests that this user sent to institutions."""
+    institutions_requested = []
+
+    queryRequests = Invite.query(Invite.sender_key == user_key,
```

Comment:
is is good, check if this invite is a request to me member, because, the actual query get all invites sent by the user, can get invites for new institutions, links and invites for new members if the user is the admin